internal/rpcproxy/rpcproxy.go

```diff
@@ -573,6 +575,9 @@ func checkPermissionsRequired(ctx context.Context, msg *message) (map[string]any
 func (p *controllerProxy) redoLogin(ctx context.Context, permissions map[string]any) error {
 	const op = errors.Op("rpc.redoLogin")
 
+	if p.anonymousLogin {
+		return errors.E(op, errors.CodeUnauthorized, "Anonymous login does not support re-authentication")
```

Comment:
Just so I'm clear... This is when the controller "anonymous logs-in", and we wouldn't redo it for permission checks because the checks in the model import flow are done at discharge? In anycase, this needs thoroughly godoc'ing because it looks really odd and is niche to offers.

Reply:
we can't do relogin because we can't gather additional facts (authorization checks) for an anonymous user..  well.. not entirely true - we could check if everyone@external has access to an application offer, for example

Reply:
Exactly as Ales has explained. I've added a godoc for the function.